### ISSUE TITLE
Is handbook supported?

### ISSUE THREAD

**Issue by myshov** [substack/stream-handbook#100]
@substack hello!
It is passed already one year since last merged pull request into master.
There are already 33 open pull requests and the same amount of issues.
Maybe is it make sense to find someone who wants to support this repo?

**Comment by dmitriz** [comment]
Would it be easy to fork it and accept the PRs in the fork?

**Comment by myshov** [comment]
@dmitriz Yes it would be easy. But I think it would be great if this repository found a second life.

**Comment by dmitriz** [comment]
@myshov 

I do my best :)

https://github.com/dmitriz/stream-handbook

**Comment by paulocoghi** [comment]
It would be even better if @substack adds @dmitriz as a second maintainer, allowing the merge from his fork to the main repo.
